meson.build

```diff
@@ -1,8 +1,10 @@
 project(
-    'dosbox-staging',
-    'c',
-    'cpp',
-    version: '0.82.0-alpha',
+    'dosbox-staging', 'c', 'cpp',
+
+    version: run_command(
+      './scripts/get-version.sh', 'version', check: true,
+    ).stdout().strip(),
```

Comment:
I can see this giving us issues but I guess we can deal with them as they come up.

1. Portability.  I would like to try at some point to get Meson running with a native Windows toolchain.  Would be straight-forward to write a Windows .bat file worse comes to worst.
2. This assumes you're running the Meson command from the project's root directory.  I can foresee CI breaking here if we make certain changes.  Suggest using `meson.project_source_root()` to be more robust (I think this is the correct one, there are several variations).

Reply:
1. We could make that work either via WSL (Windows 10 is our minimum requirement anyway), or alternatively a batch file as you suggested. It seems solvable, so we'll cross that bridge when we get there. 

2. Good tip. Changed and also improved the path handling for running the tests (see last commit).